FluentUI.Drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/sheetItem/PersistentBottomSheetContentViewProvider.kt

```diff
@@ -44,4 +50,10 @@ internal interface IViewProvider {
     fun getContentView(itemTypeList: BottomSheetParam.ItemTypeList,
```

Comment:
Should this be a separate class?
Or is it specific to just persistent bottomsheets?

Reply:
it is specific to default content builder

Reply:
it is specific to persistentbottomsheet content builder